src/eCloud/DatastoreClient.php

```diff
@@ -21,12 +49,30 @@ public function getPage($page = 1, $perPage = 15, $filters = [])
     {
         $page = $this->paginatedRequest("v1/datastores", $page, $perPage, $filters);
         $page->serializeWith(function ($item) {
-            return new Datastore($item);
+            return $this->loadEntity($item);
         });
 
         return $page;
     }
 
+    /**
+     * Expand a datastore
+     * @param $id - ID of the datastore
+     * @param $sizeGB - New size of the datastore
+     * @return bool
+     */
+    public function expand($id, $sizeGB)
```

Comment:
Does the API take into account the billing here?

Reply:
Should we also be passing the Datastore object into here instead of the size?

Reply:
Billing will be dealt with in automation. Actually, this endpoint will be locked down to admin until the billing is added there.